aten/src/ATen/native/quantized/affine_quantizer.cpp

```diff
@@ -151,13 +151,14 @@ Tensor& quantize_tensor_per_channel_affine(
 
   checkRoundingMode(fn_name);
   checkFloatTensor(fn_name, rtensor);
-  checkCPUTensor(fn_name, rtensor);
   checkSameDevice(fn_name, rtensor, qtensor);
   checkSameSize(fn_name, qtensor, rtensor);
 
   AT_DISPATCH_QINT_TYPES(qtensor.scalar_type(), fn_name, [&]() {
     checkQuantizedTensor<scalar_t>(fn_name, qtensor);
-    checkZeroPoints<underlying_t>(fn_name, zero_points);
+    if(qtensor.device().type() != c10::DeviceType::CUDA){
+      checkZeroPoints<underlying_t>(fn_name, zero_points);
```

Comment:
Does this work now for cuda tensors?

Reply:
this bypasses those checks for cuda tensors because the checkZeroPoints code doesn't work for cuda, those checks for cuda tensors are added back in here: https://github.com/pytorch/pytorch/pull/59701